tests/functions/slicer_test.py

```diff
@@ -1,46 +1,75 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
 
-from plopp.data import data_array
+from plopp.data import data_array, dataset
 from plopp.functions.slicer import Slicer
 import scipp as sc
+import pytest
 
 
 def test_creation_keep_two_dims():
     da = data_array(ndim=3)
-    sl = Slicer(da=da, keep=['xx', 'yy'])
+    sl = Slicer(da, keep=['xx', 'yy'])
     assert sl.slider.value == {'zz': 0}
     assert sl.slider.controls['zz']['slider'].max == da.sizes['zz'] - 1
-    assert sc.identical(sl.slice_node.request_data(), da['zz', 0])
+    assert sc.identical(
+        list(sl.figure.graph_nodes.values())[0].request_data(), da['zz', 0])
```

Comment:
Can we rely on the order of nodes? I.e. is the order part of the contracts of the API?

It seems to me that a relatively high level type like `Slicer` may choose an order internally and change it if that is beneficial or just as a side effect of a refactor.

Reply:
Hmm there should only be one node here, so order shouldn't matter. But in the other tests I used `sl.slice_nodes[0]` instead, which makes more sense. So I will change to that. Thanks!